torch/_functorch/_aot_autograd/utils.py

```diff
@@ -478,6 +483,9 @@ def copy_fwd_metadata_to_bw_nodes(fx_g: torch.fx.GraphModule) -> None:
         if isinstance(submod, torch.fx.GraphModule):
             _collect_fwd_nodes_from_subgraph(submod, fwd_seq_nr_to_node)
 
+    for k, v in fwd_seq_nr_to_node.items():
```

Comment:
Wrap this in a if statement with isEnabledFor(logging.DEBUG) to make this zero overhead in the common case